common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -712,11 +689,34 @@ def change_enrollment(strategy, user=None, is_dashboard=False, *args, **kwargs):
             except (
                 CourseDoesNotExistException,
                 ItemAlreadyInCartException,
-                AlreadyEnrolledInCourseException
+                AlreadyEnrolledInCourseException,
             ):
                 pass
             # It's more important to complete login than to
             # ensure that the course was added to the shopping cart.
             # Log errors, but don't stop the authentication pipeline.
-            except Exception as ex:
+            except Exception as ex:  # pylint: disable=broad-except
                 logger.exception(ex)
+
+
+@partial.partial
```

Comment:
actually, I reverted the removal.  my logic being that this is a function within the pipeline.  All pipeline functions, from what I can tell are expected to have this decorator.  If this function were ever updated in the future such that it is valid for an re-entrant entry point (other than `AUTH_ENTRY_LOGIN_API`), this missing decorator would be easily missed.